src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -54,37 +55,41 @@
                     (int (Math/ceil (layout-y %))))
                  (into #{} (map (comp :name second) channels-list)))
     :style     {:margin-top 20 :flex 1}}
-   (map
-    (fn [[category-id {:keys [chats name collapsed?]}]]
-      [rn/view
-       {:flex      1
-        :key       category-id
-        ;; on-layout fires only when the component re-renders, so
-        ;; in case the category hasn't changed, it will not be fired
-        :on-layout #(on-category-layout name (int (layout-y %)))}
-
-       (when-not (= constants/empty-category-id category-id)
-         [quo/divider-label
-          {:label            name
-           :on-press         #(collapse-category
-                               community-id
-                               category-id
-                               collapsed?)
-           :chevron-icon     (if collapsed? :main-icons/chevron-right :main-icons/chevron-down)
-           :padding-bottom   (if collapsed? 7 0)
-           :chevron-position :left}])
-       (when-not collapsed?
-         [rn/view
-          {:margin-left   8
-           :margin-top    10
-           :margin-bottom 8}
-          (map (fn [channel]
-                 [rn/view
-                  {:key        (:id channel)
-                   :margin-top 4}
-                  [quo/channel-list-item channel]])
-               chats)])])
-    channels-list)])
+   (for [[category-id {:keys [chats name collapsed?]}] channels-list]
```

Comment:
FYI: I didn't have to, but I decided to change from `map` to `for`. `map` is less readable IMO when the map function has too many lines, because it forces the reader to jump to the line at the bottom just to check what exactly is being mapped on.